linera-sdk/src/lib.rs

```diff
@@ -101,6 +101,9 @@ pub trait Contract: WithContractAbi + ContractAbi + Sized {
     /// instead.
     type InstantiationArgument: Serialize + DeserializeOwned + Debug;
 
+    /// Event values for streams created by this application.
+    type EventValue: Serialize + DeserializeOwned + Debug;
```

Comment:
How about `type StreamName` here?

Reply:
(could also be another PR)

Reply:
`StreamName`s can also be used by other applications, for subscriptions. So `process_events`, at least, will have to use `StreamId`s that only contain `Vec<u8>`, because they could be from different applications.